lib/common.h

```diff
@@ -27,8 +27,19 @@
 #include <QHostAddress>
 #include <QVector>
 #include <QMutex>
+#include <QtCore>
 #include "address.h"
 
+#define DEF_HTTP_METHOD(V) \
```

Comment:
1. This can be implemented as a regular function. 
2. No need to define it in header file, it's only used by common.cpp